loader/src/Loader.cc

```diff
@@ -452,9 +452,8 @@ namespace ignition
       // state gets cleared each time it is called.
       dlerror();
 
-      // NOTE: We open using RTLD_LOCAL instead of RTLD_GLOBAL to prevent the
-      // symbols of different libraries from writing over each other.
-      void *dlHandle = dlopen(_full_path.c_str(), RTLD_LAZY | RTLD_LOCAL);
+      // NOTE: We are using RTLD_GLOBAL, this may overwrite the symbols of different libraries.
```

Comment:
`./loader/src/Loader.cc:455:  Lines should be <= 80 characters long  [whitespace/line_length] [2]`